src/glados/tui.py

```
@@ -352,22 +417,67 @@ def start_glados(self) -> None:
+            self.notify("Instantiation already in progress!", severity="warning")
+            return
 
-        self.glados = self.run_worker(glados.start_listen_event_loop, exclusive=False, thread=True)
-        pass
+        self.instantiation_worker = self.run_worker(
+            self.instantiate_glados,  # The callable function
+            thread=True,  # Run in a thread (default)
+        )
 
     @classmethod
     def run_app(cls, config_path: str | Path = "glados_config.yaml") -> None:
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)